docs/topics/vs-code-extension-interface.adoc

```diff
@@ -0,0 +1,18 @@
+// Module included in the following assemblies:
+//
+// * docs/vsc-extension-guide/master.adoc
+
+
+[id="vs-code-extension-interface_{context}"]
+= {ProductShortName} extension interface
+
+The following components are displayed in the interface of the {ProductName} ({ProductShortName}) extension:
+
+* The Run configuration pane is to the right of the Activity bar. The Run configuration pane displays the applications and files analyzed by MTA and any issues uncovered by the analysis. The Run configuration pane includes a folder icon named *Report*, which you can click to open the MTA Report in your browser.
+
+* The Configuration settings pane, which is in the upper right pane of the extension interface, is where you configure your analyses. When an analysis begins, the lower portion of this pane is covered by the Run configuration detail pane.
+
+* The Run configuration detail pane, which is in the lower right pane of the extension interface, displays your configuration settings, such as source, target, and advanced options. During an analysis, a progress window is displayed in the Run configuration detail pane. When the analysis is completed, the progress window includes a button that opens the MTA report when clicked.
```

Comment:
Make sure it really is only used for viewing run configuration details. I prefer "details" to "detail".

Be careful about passive language.